core/src/err/mod.rs

```diff
@@ -1099,7 +1099,7 @@ pub enum Error {
 	InvalidStorageVersion,
 
 	/// There was an outdated storage version stored in the database
-	#[error("The data stored on disk is out-of-date with this version. Please follow the upgrade guides in the documentation")]
+	#[error("The data stored on disk is out-of-date with this version. Please back up your data and use the `surreal fix` command to upgrade the storage version.")]
```

Comment:
I'd rather leave this as-is, and defer to documentation. Only because if we want to give other advice, then this will be hardcoded into the binary. For instance if the steps to upgrade for local or WebAssembly are different, then this message will be wrong, and we won't be able to guide (and update as we learn) the user to the instructions that are correct for their scenario.

Reply:
Okay, makes sense!
```suggestion
	#[error("The data stored on disk is out-of-date with this version. Please follow the upgrade guides in the documentation")]
```